### ISSUE TITLE
Torgny and Wafura 

### ISSUE THREAD

**Issue by MrDolphinCodes** [sho-87/rok-talents#43]
Thank you for are your hard work! I would really appreciate Torgny and Wafura being added!

**Comment by sho-87** [comment]
are they new commanders? I dont see them on [rok.guide](https://www.rok.guide/commanders/)

**Comment by MrDolphinCodes** [comment]
Yes they are new commanders exclusively available in the new power up version of heroic anthem KvK. They are automatically expertised and level 60 upon joining the KvK and removed when you leave. I attached images of them in this message. They have a new unique talent tree called the engineering tree.
![9C7AA19E-4121-4226-9402-8F4D714E81D5](https://user-images.githubusercontent.com/106354581/187077754-e68fa205-77e1-4784-b445-02263d8701a1.jpeg)
![5F5256B0-A50B-433C-AF80-BB97C8C10003](https://user-images.githubusercontent.com/106354581/187077764-3e10d220-5fb3-4862-bf52-cc8b4af862ff.jpeg)
![FCA8C958-9C6F-4E08-AE07-36FAB47DDC86](https://user-images.githubusercontent.com/106354581/187077773-422b38e7-78b8-4bab-962b-745ea132b318.jpeg)
![E72EED3B-FB3B-44EB-A73E-8F103D2D035B](https://user-images.githubusercontent.com/106354581/187077774-ff1bc882-ffef-47fb-b37f-9540529bc43f.jpeg)

**Comment by sho-87** [comment]
ah I see. so normally adding a new commander is pretty straightforward, but in this case it would require setting up an entirely new talent tree, which is quite time consuming. will have to see if/when I have time to work on it, but not sure it'd be worth it for temporary events that only a portion of the player base would have access to

**Comment by MrDolphinCodes** [comment]
I would definitely wait at least until the Heroic Anthem Power Up version comes up again as Lilith will probably make several changes to the commanders.
